pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -417,16 +432,23 @@ func (c *ruleCache) PatchAddressGroup(patch *v1beta1.AddressGroupPatch) error {
 	c.addressSetLock.Lock()
 	defer c.addressSetLock.Unlock()
 
-	podSet, exists := c.addressSetByGroup[patch.Name]
+	groupMemberSet, exists := c.addressSetByGroup[patch.Name]
 	if !exists {
 		return fmt.Errorf("AddressGroup %v doesn't exist in cache, can't be patched", patch.Name)
 	}
 	for i := range patch.AddedPods {
```

Comment:
Question - we still keep AddedPods/RemovedPods, after add AddedGroupMembers/RemovedGroupMembers?

Reply:
The plan is to deprecate GroupMemberPod and GroupMemberPodSet in the future, so those funcs will be unified.

Reply:
But as of now when controller will send AddedPods/RemovedPod, when AddedGroupMembers/RemovedGroupMembers?